jobs/integr8ly/ocp4/osd/cluster/deprovision/Jenkinsfile

```diff
@@ -23,7 +23,48 @@ node("cirhos_rhel7") {
                     sh "ocm login --url=https://api.stage.openshift.com/ --token=${OCM_ACCESS_TOKEN}"
                 }
             } // stage
-            
+
+            stage("login to oc on BYOC clusters") {
+                script {
+                    if(params.BYOC == "true") {
```

Comment:
replace `params.BYOC == "true"` with `params. BYOC.toString() == 'true'`

Reply:
I have changed this to `if(params.BYOC)` as it's a boolean, will that work?

Reply:
it will